frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1378,66 +1441,81 @@ function getInfoMenuItem(
 }
 
 function ContextMenuInner() {
-  const props = useSelector((state: OxalisState) => {
```

Comment:
react emitted a warning because this selector always returned different object instances. therefore, I refactored this accordingly.